src/ui/util/columns/templates/Name.tsx

```diff
@@ -0,0 +1,15 @@
+import type { Player } from "../../../../common/types";
+import { PlayerNameLabels } from "../../../components";
+import type { MetaCol } from "../getCols";
+
+export default (p: Player, c: MetaCol) => (
```

Comment:
Initially the way i wanted this to work was the col.template property was a closure to render the html, but because closures cannot be cloned when sending data from the worker to the view I couldnt store the closures in `getCols`.

Will change these over thou to atleast use props

Reply:
created type `TemplateProps` which is desructured now destructured in the column templates
```
export type TemplateProps = {
	p: Player;
	c: MetaCol;
	vars: LeagueVars;
};
```